packages/contracts-frontend/README.md

```diff
@@ -0,0 +1,73 @@
+# @uma/contracts-frontend
+
+This package exports all UMA smart contract artifacts specifically meant for consumption in a frontend. These exports
+are specifically optimized for frontends so they are able to drop any unused artifacts in order to keep the bundle size
+small.
+
+## Installing the package
+
+```bash
+yarn add @uma/contracts-frontend
+```
+
+## Importing artifacts directly
+
+To import artifacts directly into your tool of choice, the index file provides convenient functions for doing so:
+
+```js
+import { getVotingAbi, getVotingBytecode, getVotingAddress } from "@uma/contracts-frontend"
+
+// Gets the abi object for the Voting contract.
+// This will be needed to interact with the voting contract.
+const votingAbi = getAbiVotingAbi()
+
+// Gets the bytecode string for the voting contract.
+// This should usually only be required if you need to deploy a voting contract (rare).
+const votingBytecode = getVotingBytecode()
+
+// Gets the address of the voting contract for chain id 1 (eth mainnet).
+// If there is no single canonical deployment of a particular contract on the network provided, this will fail.
+const chainId = 1
+const votingAddress = getVotingAddress(chainId)
```

Comment:
would it be useful to change this to default return mainnet if no `chainId` is provided?

Reply:
This is an interesting idea. As we move toward having multiple "production" networks (like polygon), it seems a little scary to let this silently default. Thoughts?

Reply:
good point. In that case explicitly requiring the user to define the net id sounds better!